themes/default/content/what-is/database-comparison-cosmos-db-vs-dynamodb.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Database Comparison: Cosmos DB vs DynamoDB"
+meta_desc: |
+     Compare noSQL databases Cosmos DB vs DynamoDB. See the similarities and differences between these databases to determine which is best for you.
+
+type: what-is
+page_title: "Database Comparison: Cosmos DB vs DynamoDB"
+
+customer_logos:
+  title: Leading engineering organizations are building with Pulumi
+  logos:
+    - items:
+      - snowflake
+      - tableau
+      - atlassian
+      - fauna
+      - ware2go
+    - items:
+      - mindbody
+      - sourcegraph
+      - fenergo
+      - skai
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+## Cosmos DB vs DynamoDB: What Are The Similarities?
+
+Both Cosmos DB and DynamoDB are NoSQL, or non-relational, databases. This means their data is stored in some format other than two-dimensional tables. Some commonly-used formats are documents, key-value pairs, graphs, and columns, each with different strengths and tradeoffs.
+
+Cosmos DB and DynamoDB are both highly-available, scalable, globally distributed and fully-managed serverless NoSQL databases. Both function as document based or key-value based databases.
+
+## Cosmos DB vs DynamoDB: What Are The Differences?
+
+The most significant difference between Cosmos DB and DynamoDB is that Cosmos DB was designed as a drop-in replacement for an enterprise-scale relational database, and thus supports queries that look very similar to traditional SQL. DynamoDB does not support SQL-style queries, although its own filters do allow you to get the similar information with a different syntax.
+
+### Pros and Cons of Cosmos DB
+
+Cosmos DB offers a high degree of flexibility, thanks to its variety of available data models and the ability to use SQL-like queries in addition to Gremlin, Azure Tables, and MongoDB APIs when interacting with your data. While DynamoDB functions only as a document or a key-value database, Cosmos DB can function as document, key-value, wide column, and graph-based database. Its five consistency models (eventually consistent, consistent prefix, session, bounded staleness, and strong consistency) allow you to fine-tune your latency, and it scales quite well. It is priced according to storage and throughput
+
+However, this flexibility can make Cosmos DB somewhat more difficult to use at first. Understanding the various data models available to you and the intended use case of each one is key to ensuring that you are getting the most optimum performance for the price, according to your application's specific needs.
+
+### Pros and Cons of DynamoDB
```

Comment:
Same question here regarding whether it's worth mentioning somewhere in here that it's AWS specific.